libs/designsystem/src/lib/components/avatar/avatar.component.scss

```diff
@@ -46,7 +46,11 @@ $badge-diameter: utils.$avatar-badge-size;
   --kirby-icon-font-size: #{utils.icon-font-size('sm')};
 
   &.shadow {
```

Comment:
This is the class you should remember to remove, when you have refactored the class binding in `avatar.component.html` :)

Reply:
removed!